src/Microsoft.AspNetCore.Localization/CookieRequestCultureProvider.cs

```diff
@@ -2,9 +2,14 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
 
 using System;
+using System.Globalization;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Internal;
+using Microsoft.Extensions.Localization.Internal;
```

Comment:
I use it for the `ParsedCulture()` extension method, we already have something similar [here](https://github.com/aspnet/Localization/blob/dev/src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizer.cs#L10)  for `SearchedLocation()`